CactbotOverlay/CactbotEventSource.cs

```diff
@@ -189,10 +189,12 @@ public override void SaveConfig(IPluginConfig config)
         LogInfo("Language: {0}", language_);
       }
 
-      // Temporarily target cn if plugin is old v2.0.4.0
-      if (language_ == "cn" || ffxiv.ToString() == "2.0.4.0") {
+      if (language_ == "cn") {
```

Comment:
I removed the hard check on the version string since new releases of the FFXIV Plugin have cn support.

Reply:
But does the ffxiv plugin that the cn and ko players are using have that yet? I thought they were all using an older version.

Reply:
Oh, I was falsely assuming they were backwards-compatible without really thinking it through. I'll re-add that line for cn for now.

I wonder if there's anyone who knows which version of the plugin the Korean players are using?

Reply:
@devunt Hi, can you help this about the Korean act plugin version?

Reply:
@devunt Hi~ Can you help with this Korean act plugin version problem?

Reply:
I talked to @hibiyasleep who said that they used the 2.0.4.0 version directly (with no translations).  I'm not sure that there's any way to differentiate the language then until the ffxiv plugin is new enough to include cn and ko language options.

So, maybe this is good enough for now.